network-api/networkapi/wagtailpages/pagemodels/buyersguide/forms.py

```diff
@@ -0,0 +1,35 @@
+from wagtail.admin.forms import WagtailAdminModelForm, WagtailAdminPageForm
+
+
+class BuyersGuideProductCategoryForm(WagtailAdminModelForm):
```

Comment:
Like mentioned in other comment, I made no updates to `BuyersGuideProductCategoryForm`, other than moving it to this file